dask_cuda/get_device_memory_objects.py

```diff
@@ -140,3 +142,15 @@ def sizeof_cupy_base_memory(x):
     @sizeof.register(cupy.ndarray)
     def sizeof_cupy_ndarray(x):
         return int(x.nbytes)
+
+
+@sizeof.register_lazy("pylibcudf")
+def register_pylibcudf():
+    import cupy
```

Comment:
Oh, cupy isn't a (required) dependency. I'll find another way to get nbytes out of this.